modules/nf-core/bakta/bakta/tests/main.nf.test

```diff
@@ -0,0 +1,83 @@
+nextflow_process {
+
+    name "Test Process BAKTA_BAKTA"
+    script "../main.nf"
+    config "./nextflow.config"
+    process "BAKTA_BAKTA"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "bakta"
+    tag "bakta/bakta"
+    tag "bakta/baktadbdownload"
+
+    test("Bakta - bacteroides_fragilis - genome.fasta") {
+
+        setup {
+            run("BAKTA_BAKTADBDOWNLOAD") {
+                script "../../baktadbdownload/main.nf"
+                process {
+                    """
+                    """
+                }
+            }
+        }
+
+        when {
+            process {
+                """             
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.test_data['bacteroides_fragilis']['illumina']['test1_contigs_fa_gz'], checkIfExists: true)
+                ]
+                input[1] = BAKTA_BAKTADBDOWNLOAD.out.db
+                input[2] = []
+                input[3] = []
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert path(process.out.embl.get(0).get(1)).exists() },
+                { assert path(process.out.faa.get(0).get(1)).exists() },
+                { assert path(process.out.ffn.get(0).get(1)).exists() },
+                { assert path(process.out.fna.get(0).get(1)).exists() },
+                { assert path(process.out.gbff.get(0).get(1)).exists() },
+                { assert path(process.out.gff.get(0).get(1)).exists() },
+                { assert path(process.out.hypotheticals_tsv.get(0).get(1)).exists() },
+                { assert path(process.out.hypotheticals_faa.get(0).get(1)).exists() },
+                { assert path(process.out.tsv.get(0).get(1)).exists() },
+                { assert path(process.out.txt.get(0).get(1)).exists() },
```

Comment:
Really? Nothing is consistent, nor has any stable strings that can be used to confirm the file isn't just empty?

Reply:
Right, the pytest didn't check for content as well, thanks for questioning that. There are timestamps, but also other reliable strings! Will update.

Reply:
Huh ok, well strings best if possible :)